dg-core/src/main/java/org/finra/datagenerator/consumer/DataPipe.java

```diff
@@ -86,4 +90,74 @@ public String getPipeDelimited(String[] outTemplate) {
 
         return b.toString();
     }
+
+    /**
+     * Given an array of variable names, returns a JsonObject
+     * of values.
+     *
+     * @param outTemplate an array of {@link String}s containing the variable
+     * names.
+     * @return a json object of values
+     */
+    public JSONObject getJsonFormatted(String [] outTemplate) {
```

Comment:
@mauricio1990silva can we move these formatting functions out of DataPipe? Even the one for DefaultWriter (pipe-delimited) it was probably cleaner to have in the writer itself.

Disclaimer: I am the one who put the pipe-delim function in here :)